### ISSUE TITLE
too small slice for this size

### ISSUE THREAD

**Issue by hotid** [xssnick/tonutils-go#2]
client := liteclient.NewClient()

	err := client.Connect(context.Background(),
		"67.207.74.182:4924",
		"peJTw/arlRfssgTuf9BMypJzqOi7SXEqSPSWiEw2U1M=")
	if err != nil {
		panic(err)
	}
	api := ton.NewAPIClient(client)
	block, err := api.GetBlockInfo(context.Background())
	if err != nil {
		Log.Fatalln("get block err:", err.Error())
		return
	}

	_, err = api.RunGetMethod(context.Background(), block, address.MustParseAddr("EQCunKWfEsKkdhSXZAEWDpck8rSvcIjV_V_6CBA6UsI0XRBu"), "get_nft_data")
	if err != nil {
		Log.Errorf("run get method err: %s", err.Error())
		return
	}

>     panic: too small slice for this size
>     goroutine 1 [running]:
>     github.com/xssnick/tonutils-go/tvm/cell.(*Builder).MustStoreSlice(...)
>             .../github.com/xssnick/tonutils-go@v0.1.2/tvm/cell/builder.go:134
>     github.com/xssnick/tonutils-go/ton.(*APIClient).RunGetMethod(0xc000119f40, {0x70aec8, 0xc000016050}, 0xc000194140, 0xc0001820a0, {0x6d1222, 0xc}, {0x0, 0x0, 0x0})
>             .../github.com/xssnick/tonutils-go@v0.1.2/ton/api.go:253 +0x1745
>     main.main()
>             .../test/lite-client.go:29 +0x205

    > runmethod EQCunKWfEsKkdhSXZAEWDpck8rSvcIjV_V_6CBA6UsI0XRBu get_nft_data
    [ 3][t 1][2022-04-29 00:52:24.707931189][lite-client.cpp:1324][!testnode]       requesting remote get-method execution for 0:AE9CA59F12C2A47614976401160E9724F2B4AF7088D5FD5FFA08103A52C2345D with respect to (-1,8000000000000000,20168059):A348D3D666FF09B2FAEC3F06419F7D9CA9F1B6CC4FF62D0E865C8040D05994CB:F14E06284BC7A8F525EC1BE4210A7D56EE9FECF58AB8466BB6F48D2FAB75D489 to run method get_nft_data with 0 parameters
    [ 3][t 2][2022-04-29 00:52:24.767547252][lite-client.cpp:1998][!testnode]       got (partial) account state (637 bytes) with mode=31 for 0:AE9CA59F12C2A47614976401160E9724F2B4AF7088D5FD5FFA08103A52C2345D with respect to blocks (-1,8000000000000000,20168059):A348D3D666FF09B2FAEC3F06419F7D9CA9F1B6CC4FF62D0E865C8040D05994CB:F14E06284BC7A8F525EC1BE4210A7D56EE9FECF58AB8466BB6F48D2FAB75D489 and (0,8000000000000000,25326017):6FEB8CDE0C810A3CBC4FEEC5685103FD5FCEF48B7DDCEAF320668D153369796D:6DA394AC82E92EAD9F9770796F9260CB5B3C2CD2A91AEA2DEB075C5CC8AA6BF9
    [ 3][t 2][2022-04-29 00:52:24.767826707][lite-client.cpp:2095][!testnode]       starting VM to run method `get_nft_data` (102351) of smart contract 0:AE9CA59F12C2A47614976401160E9724F2B4AF7088D5FD5FFA08103A52C2345D
    arguments:  [ 102351 ]
    result:  [ -1 0 CS{Cell{0295000000000000000080051eec1b065126304253957c14810c949f9902ec722d1aa47adb690497b32f7cd0027e457611181313ddf9812a3ef5d3b9dcf80c7f1bdb9ff8d57fc9e74a3ce93c46} bits: 64..331; refs: 0..0} CS{Cell{0295000000000000000080051eec1b065126304253957c14810c949f9902ec722d1aa47adb690497b32f7cd0027e457611181313ddf9812a3ef5d3b9dcf80c7f1bdb9ff8d57fc9e74a3ce93c46} bits: 331..598; refs: 0..0} C{62EDF0EE80EF8562B380870C568CB4965EBED8FFF0308548C263FB9FAD023020} ]
    remote result (not to be trusted):  [ -1 0 CS{Cell{0295000000000000000080051eec1b065126304253957c14810c949f9902ec722d1aa47adb690497b32f7cd0027e457611181313ddf9812a3ef5d3b9dcf80c7f1bdb9ff8d57fc9e74a3ce93c46} bits: 64..331; refs: 0..0} CS{Cell{0295000000000000000080051eec1b065126304253957c14810c949f9902ec722d1aa47adb690497b32f7cd0027e457611181313ddf9812a3ef5d3b9dcf80c7f1bdb9ff8d57fc9e74a3ce93c46} bits: 331..598; refs: 0..0} C{62EDF0EE80EF8562B380870C568CB4965EBED8FFF0308548C263FB9FAD023020} ]
    [ 3][t 2][2022-04-29 00:52:24.767973694][lite-client.cpp:1225][!testnode]       caching cell 62EDF0EE80EF8562B380870C568CB4965EBED8FFF0308548C263FB9FAD023020

**Comment by xssnick** [comment]
Hey @hotid, thank you for reporting, fixed, please pull v0.1.3 tag (latest)

`RESP UINT 18446744073709551615
RESP UINT 0
RESP CELL 596 bits, hex(000000000000000080051eec1b065126304253957c14810c949f9902ec722d1aa47adb690497b32f7cd0027e457611181313ddf9812a3ef5d3b9dcf80c7f1bdb9ff8d57fc9e74a3ce93c40)
RESP CELL 596 bits, hex(000000000000000080051eec1b065126304253957c14810c949f9902ec722d1aa47adb690497b32f7cd0027e457611181313ddf9812a3ef5d3b9dcf80c7f1bdb9ff8d57fc9e74a3ce93c40)
RESP CELL 0 bits, hex()`

**Comment by xssnick** [comment]
Oh, looks like found 1 more issue with bits, at the end of cell: 46 vs 40, will check

**Comment by xssnick** [comment]
Interesting, maybe its not bug actually :D 

From lite-client log which you attached I see
```
02 95 000000000000000080051eec1b065126304253957c14810c949f9902ec722d1aa47adb690497b32f7cd0027e457611181313ddf9812a3ef5d3b9dcf80c7f1bdb9ff8d57fc9e74a3ce93c46 
```

0x02 means references number
0x95 means length of octets (bits * 4), in dec system it will equal 149, 149 octets = 596 bits which will equal 
```
000000000000000080051eec1b065126304253957c14810c949f9902ec722d1aa47adb690497b32f7cd0027e457611181313ddf9812a3ef5d3b9dcf80c7f1bdb9ff8d57fc9e74a3ce93c40
```
which is actual result of this library. (0x40 at the end)

Maybe its lite client hex representation bug, but I will try to clarify it.

So IMO, you can use it safely for now, I'll write here result of research later.

**Comment by xssnick** [comment]
Fixed cell issue in v0.1.4
